dependency_manager/src/edm_tool/edm.py

```diff
@@ -1165,6 +1165,9 @@ def release_handler(args):
 
     cpm_modules_path = build_path / "CPM_modules"
     everest_core_repo_info = GitInfo.get_git_repo_info(everest_core_path)
+    everest_core_repo_info_git_tag = "unknown"
+    if everest_core_repo_info["rev"]:
```

Comment:
is it possible by the program logic, that 'rev', 'branch' and/or 'tag' are set at the same time? If so, I would use an `if else` here.

Reply:
Thats actually desired and only the most descriptive one should be used then, so if there's a rev, branch and tag available if would always pick the tag, if there's a rev an tag it would also pick the tag and if there's only a rev it would include that tag as a version component. So I think this works as intended here

Reply:
right, completely fine :1234: